billing.go

```diff
@@ -0,0 +1,76 @@
+package easypost
+
+import (
+	"context"
+	"errors"
+)
+
+// DeletePaymentMethod allows you to delete a payment method in your wallet.
+func (c *Client) DeletePaymentMethod(priority PaymentMethodPriority) (err error) {
+	return c.DeletePaymentMethodWithContext(context.Background(), priority)
+}
+
+// DeletePaymentMethodWithContext performs the same as DeletePaymentMethod, but
+// allows specifying a context that can interrupt the request.
+func (c *Client) DeletePaymentMethodWithContext(ctx context.Context, priority PaymentMethodPriority) (err error) {
+	paymentMethodObject, _ := c.getPaymentMethodByPriority(priority)
+	endpoint, _ := c.getPaymentMethodEndpoint(paymentMethodObject)
+	return c.del(ctx, endpoint+"/"+paymentMethodObject.ID)
+}
+
+// FundWallet allows you to fund your EasyPost wallet by charging your primary or secondary payment method on file.
+func (c *Client) FundWallet(amount string, priority PaymentMethodPriority) (err error) {
+	return c.FundWalletWithContext(context.Background(), amount, priority)
+}
+
+// FundWalletWithContext performs the same as FundWallet, but
+// allows specifying a context that can interrupt the request.
+func (c *Client) FundWalletWithContext(ctx context.Context, amount string, priority PaymentMethodPriority) (err error) {
+	paymentMethod, _ := c.getPaymentMethodByPriority(priority)
+	endpoint, _ := c.getPaymentMethodEndpoint(paymentMethod)
+
+	wrappedParams := map[string]interface{}{
+		"amount": amount,
+	}
+
+	out := PaymentMethodObject{}
+	err = c.post(ctx, endpoint+"/"+paymentMethod.ID+"/charges", wrappedParams, &out)
+	return err
```

Comment:
```suggestion
	return
```

I don't think you need this since you already specify the return value in line 28.